config/archunit-store/stored.rules

```diff
@@ -0,0 +1,3 @@
+#
+#Sat Sep 10 22:27:26 IST 2022
```

Comment:
Sorry one more thing I just noticed, do we really need this date? We won't be updating it right?

Reply:
No, I don't think so, these are comments most proly, so I removed them.